### PR TITLE
Few more items for 1.0.10

### PR DESCRIPTION
**Pull request by Spuds**

### CODE REVIEW

File: themes/default/css/_besocial/jquery.sceditor.elk_wiz_besocial.css [hunk 1]
```diff
@@ -1,8 +1,19 @@
 html, body, p, code:before, table {
 	margin: 0;
-	padding: 0;
+	padding: 0 0 0 0;
```
Comment (live627) [inline themes/default/css/_besocial/jquery.sceditor.elk_wiz_besocial.css]: !

Comment (Spuds) [inline themes/default/css/_besocial/jquery.sceditor.elk_wiz_besocial.css]: correct

Comment (Spuds) [inline themes/default/css/_besocial/jquery.sceditor.elk_wiz_besocial.css]: !!

Comment (Trekkie101) [inline themes/default/css/_besocial/jquery.sceditor.elk_wiz_besocial.css]: !!!!;

**Comment by Spuds** [comment]
Not sure whats up with SensioLabsInsight

**Comment by emanuele45** [comment]
I wouldn't care about it to much, it frequently stops for not that much... :-\
I restarted it, but it's not really critical to me. :smiley: 

Is that one ready to merge?

**Comment by Spuds** [comment]
Should be all set now, not sure why my less compiler added the 0's but hey ho there you go

**Pull request merged**